src/components/Carousel.jsx

```diff
@@ -0,0 +1,57 @@
+import React,{useState} from 'react';
+import icare1 from '../assets/icare1.jpg';
+import icare2 from '../assets/icare2.jpg';
+import icare3 from '../assets/icare3.jpg';
+import icare4 from '../assets/icare4.jpg';
+import icare5 from '../assets/icare5.jpg';
+import {ChevronLeft,ChevronRight} from 'react-feather';
+export default function Carousel() {
+const slides=[
+    {
+        src:icare1,
+    },
+    {
+        src:icare2,
+    },
+    {
+        src:icare3,
+    },
+    {
+        src:icare4,
+    },
+    {
+        src:icare5,
+    }
+];
+const [current,setCurrent]=useState(0);
+const length=slides.length;
+const nextSlide=()=>{
+    setCurrent(current===length-1?0:current+1);
+};
+const prevSlide=()=>{
+    setCurrent(current===0?length-1:current-1);
+};
+if(!Array.isArray(slides)||slides.length<=0){
+    return null;
+}
+
+return (
+    <section className="carousel">
```

Comment:
Is this class being used anywhere? If not, remove
Instead give it an id and also add the link to the Navbar